dali/test/python/operator_2/test_random_beta.py

```diff
@@ -99,10 +99,7 @@ def pipeline():
         for _ in range(batch_size)
     ]
     assert len(batch) == len(refs)
-    pvs = [
-        sp.stats.kstest(sample, sample_ref, nan_policy="raise")[1]
-        for sample, sample_ref in zip(batch, refs)
-    ]
+    pvs = [sp.stats.kstest(sample, sample_ref)[1] for sample, sample_ref in zip(batch, refs)]
```

Comment:
It is OK, if no NaNs are expected in sample data, otherwise data the kstest will behave differently.

Reply:
We should be good with the change. The NaNs are not expected here and so the test should fail if there are some. The line is followed with `>=` checks expected to be true, so they will fail on NaNs. I just hoped for early error and cleaner messages.